### ISSUE TITLE
IdentityLookup support for Go cluster

### ISSUE THREAD

**Issue by rogeralsing** [asynkron/protoactor-go#408]
https://github.com/AsynkronIT/protoactor-dotnet/blob/dev/src/Proto.Cluster/IIdentityLookup.cs

This should be ported to Go.

See the MongoIdentityLookup as a reference source on how this works
https://github.com/AsynkronIT/protoactor-dotnet/tree/dev/src/Proto.Cluster.MongoIdentityLookup

**Comment by umitkavala** [comment]
I'd like to start this one but I couldn't figured it out some needs. 
For instance  I need this type:
```
type Cluster struct {
	ActorSystem    *actor.ActorSystem
	Config         *ClusterConfig
	remote         *remote.Remote
	pidCache       *pidCacheValue
	MemberList     *memberListValue
	partitionValue *partitionValue
}
```
but this is in progress in #407. How do you handle this dependencies. Wait for it or work in same branch?

**Comment by umitkavala** [comment]
Got answer from @rogeralsing on Slack.

**Comment by umitkavala** [comment]
@rogeralsing could you please update the link for Mongo project?
https://github.com/AsynkronIT/protoactor-dotnet/tree/dev/src/Proto.Cluster.Identity.MongoDb

**Comment by umitkavala** [comment]
I've update protos.proto to have ClusterIdentity and other messages. When I try to build I got this error.
![image](https://user-images.githubusercontent.com/3861934/99590105-33eba000-29ed-11eb-88a6-cd6eea8ee934.png)
have any idea? (Windows 10)

**Comment by LostLaser** [comment]
@umitkavala I had the same issue when I was trying to initially setting up the project a few days ago. I did eventually just switch to working on my linux VM but I was able to get the project to run in Windows before I did so. Here are some of the steps I remember doing:

- I'll start with the largest change in that I eventually switched to just using the cygwin terminal to correct a couple of errors
- Manually "go get" the protobuf go package to address the second error line. After running the get command I would suggest a "make clean" and then another "make" to hopefully clear out those errors. 
- Your "make clean" command will likely fail if you haven't ran it before. You will get an error with the "sed" command so you will need to remove the empty single tick quotes from the command in order for it to work.
- The protopath command line arg is not getting set appropriately. The command is ignoring the backslashes in your input path.

Let me know if anyone of my steps are confusing or worded poorly.
